spec/views/collections/_sort_and_per_page.html.erb_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'collections/_sort_and_per_page.html.erb', type: :view do
+ let(:collection) { double }
+  let(:response) { double(response: { 'numFound' => 3 }) }
+  let(:search_state) { double('SearchState', params_for_search: {}) }
+
+  before do
+    allow(view).to receive(:search_state).and_return(search_state)
+    allow(view).to receive(:sort_fields).and_return(['title_sort', 'date_sort'])
+    allow(view).to receive(:document_index_views).and_return(list: Blacklight::Configuration::ViewConfig.new)
+    assign(:response, response)
+  end
+
+  context "when the action is show" do
+    before do
+      controller.action_name = "show"
+    end
+    it "renders the form without a button" do
+      expect(view).not_to receive(:button_for_remove_selected_from_collection)
```

Comment:
I don't know that this expectation is falsifiable.  Is the view expected to receive a call to `:button_for_remove_selected_from_collection`  when the action is "edit" ?